nexus/reconfigurator/execution/src/omicron_zones.rs

```diff
@@ -259,26 +278,31 @@ mod test {
         s2.verify_and_clear();
 
         // Add an `InternalNtp` zone for our next update
-        fn append_zone(zones: &mut OmicronZonesConfig) {
+        fn append_zone(zones: &mut BlueprintZonesConfig) {
             zones.generation = zones.generation.next();
-            zones.zones.push(OmicronZoneConfig {
-                id: Uuid::new_v4(),
-                underlay_address: "::1".parse().unwrap(),
-                zone_type: OmicronZoneType::InternalNtp {
-                    address: "::1".into(),
-                    dns_servers: vec!["::1".parse().unwrap()],
-                    domain: None,
-                    ntp_servers: vec!["some-ntp-server-addr".into()],
+            zones.zones.push(BlueprintZoneConfig {
+                config: OmicronZoneConfig {
+                    id: Uuid::new_v4(),
+                    underlay_address: "::1".parse().unwrap(),
+                    zone_type: OmicronZoneType::InternalNtp {
+                        address: "::1".into(),
+                        dns_servers: vec!["::1".parse().unwrap()],
+                        domain: None,
+                        ntp_servers: vec!["some-ntp-server-addr".into()],
+                    },
                 },
+                // XXX: NotInService retains the previous test behavior -- we
+                // may wish to change this to InService.
+                disposition: BlueprintZoneDisposition::Quiesced,
```

Comment:
Can we change this one to `InService` too?

Reply:
Done. I guess we should test that quiesced zones get deployed too, though.

Reply:
(so I've expanded the test to make it so that one zone is in-service and the other is quiesced -- in the future, we can also add a test for expunged.)